tests/protractor/helper.js

```diff
@@ -64,14 +65,15 @@ module.exports = {
       .catch(() => {
         browser.sleep(1000);
         return browser.wait(EC.elementToBeClickable(element), 12000).then(() => {
+          utils.handleModal();
```

Comment:
Again, don't just handle modal every time anything is clicked - the modal _only_ occurs when we change the settings. Catch and handle it there and nowhere else.